src/App.jsx

```diff
@@ -14,70 +16,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood === '' ? 'No goods selected' : `${selectedGood} is selected`}
+        {selectedGood !== ''
+          && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => setSelectedGood('')}
+            />
+          )}
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((item) => {
+            const flag = selectedGood === item;
```

Comment:
It would be great to provide a more descriptive name for boolean
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/d5d39c1e-898b-4401-bd7a-abf3c7fab3a5)